src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -632,27 +641,77 @@ export const Deposit: React.FC<DepositProps> = ({
       contextDispatch({ type: ThorchainSaversDepositActionType.SET_LOADING, payload: true })
       setIsSendMax(percent === 1)
 
+      const _percentageCryptoAmountBaseUnitBeforeTxFees = bnOrZero(balanceCryptoBaseUnit)
```

Comment:
Can leverage this base unit flavour below to keep things DRY

Reply:
Reverted in [c0cca3f](https://github.com/shapeshift/web/pull/7606/commits/c0cca3f5b1d2711ee907ab0deea35f2cbd0dbe79)